$NetBSD: patch-ta,v 1.1 2010/09/01 09:23:18 wiz Exp $

patch from upstream cvs repository to allow exmh to function
correctly with tcl 8.5 (or any earlier version).  This patch
will not be needed after the next exmh upgrade.


--- exmh-2.7.2/lib/html.tcl	2004-07-28 13:21:07.000000000 +0700
+++ lib/html.tcl	2010-08-27 03:50:58.000000000 +0700
@@ -209,7 +209,7 @@
 }
 proc HtmlHit {win x y} {
     upvar #0 HM$win var
-    lassign {href name} [UrlGetLink $win $x $y]
+    exmh_lassign {href name} [UrlGetLink $win $x $y]
     UrlResolve $var(S_url) href
     URI_StartViewer $href
 }
